packages/@react-aria/disclosure/src/useDisclosure.ts

```diff
@@ -24,7 +25,9 @@ export interface AriaDisclosureProps {
   /** Whether the disclosure is expanded (controlled). */
   isExpanded?: boolean,
   /** Whether the disclosure is expanded by default (uncontrolled). */
-  defaultExpanded?: boolean
+  defaultExpanded?: boolean,
+  /** Unique id for the disclosure. Useful if used in a disclosure group. */
```

Comment:
```suggestion
  /** Unique id for the disclosure. Required in disclosure groups. */
```
Or "Used in..." if it's not required

if it's required in DisclosureGroups though, maybe it's a good idea to make it always required? or if the DisclosureGroup context is detected, throw and error if an id isn't provided.